jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -254,6 +254,14 @@ static long getTimeStamp(Request request)
     // TODO: see above.
     boolean isSecure();
 
+    /**
+     * Signal the intention to read the request content for the purposes of 100-Continues processing.
+     * @see HttpStream#willRead()
+     */
+    default void willRead()
```

Comment:
I think this shouldn't be marked as `default`.

Reply:
ditto